crates/bevy_app/README.md

```diff
@@ -0,0 +1,5 @@
+# Bevy App
+
+[Bevy App](https://github.com/bevyengine/bevy/tree/main/crates/bevy_app) is a component of the [Bevy Engine](https://bevyengine.org/). You can revise the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).
+
+This crate is intended to be used as a part of [Bevy](https://crates.io/crates/bevy) and not independently. However, feel free to use it if it fits your needs.
```

Comment:
not true for `bevy_app`, it can be used independently

Reply:
Right, I guess `bevy_app` with `bevy_ecs` is perfectly valid.

Reply:
Ok, I remove it